projects/azure/README.md

```diff
@@ -256,3 +256,50 @@ token-refresh                             0 0 * * *   False     1         44d
 * _token-refresh_ refreshes a token from the AWS registry that we put inside node images built from Origin master to pull images for the cluster
 
 We need to check whether the above jobs are in a working state. Access to the cluster is granted to every member of the openshift github organization. Access to the azure namespace is controlled by the [azure-team group](./cluster-wide.yaml).
+
+# Building container images
+
+To onboard a new container image into CI and possibly setup mirroring of the image to quay the following steps should be 
+performed:
+
+* Ensure there is a `Dockerfile.<image_name>` in the [openshift on azure repo](https://github.com/openshift/openshift-azure) 
+specifying how to build the new image
+* Create 3 new make targets in that repository for the various stages leading up to building the image
+  * one target for building the binary
+  * one target for building the container image
+  * one target for pushing the container image to the CI registry
```

Comment:
The image is not pushed in the CI registry but quay, right?

Reply:
Right! That should be to quay.